### ISSUE TITLE
Update onboarding with ads regions

### ISSUE THREAD

**Issue by deeppandya** [brave/browser-android-tabs#2377]
## Description 
This ticket is to update onboarding changes with core changes related to ads region. It will make desktop app and android in sync for ads regions.

## Steps to reproduce 

   1. 
   2. 
   3. 

## Expected result
Desktop app and android app should be in sync for ads regions.

## Issue happens on 
   - Current Play Store version?
   - Beta build?

## Device details
   - Install type (ARM, x86):
   - Device (Phone, Tablet, Phablet):
   - Android version:
   
## Brave version
All

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by anthonypkeane** [comment]
@srirambv this is a new issue, can you pls take a look? Thanks

**Comment by samartnik** [comment]
This is included in 1.6.1

**Comment by btlechowski** [comment]
Verification passed on BraveModernarm v1.6.1-rc.2 on Samsung s7 (Android 8.0).
Regions tested:
```
      { "LT", false },  // Lithuania - ad was shown
      { "MA", false },  // Morocco - pass
      { "MY", false },  // Malaysia - ad was shown
      { "NG", false },  // Nigeria - ad was shown
      { "NO", false },  // Norway - pass
      { "PK", false },  // Pakistan - pass
      { "PT", false },  // Portugal - pass
      { "RO", false },  // Romania - pass
      { "RS", false },  // Serbia - pass
      { "RU", false },  // Russian Federation - pass
      { "SA", false },  // Saudi Arabia - ad was shown
      { "SI", false },  // Slovenia - pass
      { "SK", false },  // Slovakia - pass
      { "TH", false },  // Thailand - pass
      { "TR", false },  // Turkey - pass
      { "TW", false },  // Taiwan - pass
      { "UA", false },  // Ukraine - pass
      { "VN", false }   // Vietnam - pass
```
